src/components/FileMetadata/index.spec.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+
+import { testVersionFile } from '../../test-helpers';
+import styles from './styles.module.scss';
+
+import FileMetadata from '.';
+
+describe(__filename, () => {
+  it('renders metadata for a file', () => {
+    const file = testVersionFile;
```

Comment:
We could probably use a `store` and dispatch an action to retrieve this `file`, instead of having another object in `test-helpers`. This should be slightly more robust, and less weird.

I like that our `fake*` objects are API representations and the rest requires either `createInternal*` or an action when we need a selector.